test_xdb/models/under_test/clone_schema_test.sql

```diff
@@ -0,0 +1,200 @@
+{{config({
+    "tags":["exclude_bigquery", "exclude_bigquery_tests"],
+    "pre-hook": [{"sql": "DROP SCHEMA IF EXISTS clone_schema_one CASCADE;
```

Comment:
Here we are specifying test schemas and performing `clone_schema()` macro on them.